packages/rust/compile.tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../configs/base.tsconfig",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "lib"
+  },
+  "include": [
+    "src"
+  ],
+  "files": [
+    "../../node_modules/monaco-editor-core/monaco.d.ts"
```

Comment:
FYI did you see this? The java plugin seems to be able to contribute to monaco without requiring this

Reply:
I think it is because it imports `@theia/monaco` which has a monaco.d.ts.
A better way is to use an import. See https://github.com/theia-ide/theia/blob/master/packages/typescript/src/browser/monaco.d.ts